user/migrations/0002_alter_user_managers_remove_user_username_and_more.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.1 on 2024-12-15 14:35
+
+from django.db import migrations, models
+import user.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("user", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name="user",
+            managers=[
+                ("objects", user.models.UserManager()),
```

Comment:
Ensure that the `UserManager` class is correctly defined in `user.models` to handle user creation and management using email instead of username.